apps/src/applab/designProperties.jsx

```diff
@@ -16,6 +20,16 @@ var DesignProperties = module.exports = React.createClass({
       return <p>{applabMsg.designWorkspaceDescription()}</p>;
     }
 
+    // We want to have a unique key that doesn't change when the element id
```

Comment:
Ah, I see you are only assigning keys to elements once you've edited them. great insight! this is a much simpler solution than I had imagined.